sentry_streams/sentry_streams/pipeline/msg_parser.py

```diff
@@ -1,21 +1,31 @@
 import json
-from typing import Any
+from typing import Any, MutableMapping
 
-from sentry_streams.pipeline.message import Message
+from sentry_kafka_schemas import get_codec
+from sentry_kafka_schemas.codecs import Codec
+
+from sentry_streams.pipeline.message import Message, PyRawMessage
 
 # TODO: Push the following to docs
 # Standard message decoders and encoders live here
 # These are used in the defintions of Parser() and Serializer() steps, see chain/
 
+CODECS: MutableMapping[str, Codec[Any]] = {}
```

Comment:
Is the idea here that we're going to build our own Codecs?

Reply:
This is mostly to take the codec out of the message to avoid creating the codec each time we need to use and reuse the instances. 

But yes we will need an abstract codec that can wrap the sentry-kafka-shemas ones.